applications/convection_diffusion_application/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -164,11 +162,7 @@ def __init__(self, model, custom_settings):
 
         self.print_on_rank_zero("::[ConvectionDiffusionBaseSolver]:: ", "Construction finished")
 
-    def AddVariables(self, target_model_part=None):
-
-        if target_model_part == None:
```

Comment:
If I'm not wrong, this was required for the CHT solver. Are you sure of this change?